mobile_app_connector/data/default_hub.xml

```diff
@@ -53,7 +53,7 @@
         <record id="tile_gi7" model="mobile.app.tile">
             <field name="name">Awaiting payment</field>
             <field name="subtype_id" ref="tile_subtype_gi7"/>
-            <field name="mode">one</field>
+            <field name="mode">many</field>
```

Comment:
This is a good fix. You can keep this one and remove all others :)

Reply:
Done, thank you !